components/layout_2020/flow/mod.rs

```diff
@@ -242,10 +242,9 @@ impl BlockFormattingContext {
 
         IndependentLayout {
             fragments: flow_layout.fragments,
-            content_block_size: (flow_layout.content_block_size +
+            content_block_size: Au::from(flow_layout.content_block_size) +
```

Comment:
Can this just be `flow_layout.content_block_size.into()`?

Reply:
Due to compiler errors with .into(), which could not infer the correct type conversion, I've used Au::from(...) for explicit and clear type conversion.